tests/ui/test_ui_tools.py

```diff
@@ -1936,25 +1936,20 @@ def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture, widget_size,
         ("""<blockquote>
                 <p><a href='https://chat.zulip.org/'</a>czo</p>
             </blockquote>""", "{} czo [1]\n"),
-        pytest.param("""<blockquote>
-                            <blockquote>
-                                <p>A<br>
-                                B</p>
-                            </blockquote>
-                        </blockquote>
-            """, "{} {} A\n{} {} B",
-                     marks=pytest.mark.xfail(reason="rendered_bug")),
-        pytest.param("""<blockquote>
-                            <blockquote>
-                                <p>A</p>
-                            </blockquote>
-                            <p>B</p>
-                            <blockquote>
-                                <p>C</p>
-                            </blockquote>
-                        </blockquote>
-        """, "{} {} A\n{} B\n{} {} C",
-                     marks=pytest.mark.xfail(reason="rendered_bug")),
+        ("""<blockquote>
+                <blockquote>
+                    <p>A<br/>B</p>
```

Comment:
A quick test indicated the server actually generates a `<br>` (not xml) and after both A and B too?

There are also extra newlines in each blockquote associated with ::before and ::after - I've not looked at this in a while but are we relying on those to be present for spacing purposes?

Reply:
There's a similar test just above this, that had `<br/>` as well, so I didn't cross-check with the server.

Yes, I presume they are there because the quoting expects some text after the quote, hence the `newline`?